text/0000-reseeding-rand.md

```diff
@@ -0,0 +1,657 @@
+- Start Date: 2013-01-23
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Stabilise the `std::rand` module by focusing it on a smaller set of
+functionality, moving "fancy" functionality to a crates.io crate.
```

Comment:
That was not my immediate plan (which was just move everything not included here into a `rand` crate), but it would also work; in any case, the details of the code outside `std` aren't so important at the moment. :smile: